test/Directory.Packages.props

```diff
@@ -2,5 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
   <ItemGroup>
     <PackageVersion Update="System.Text.Json" Version="6.0.5" />
+    <PackageVersion Include="Microsoft.Coyote" Version="1.7.10" />
```

Comment:
Do you need to use both the packages?

Reply:
Good catch - we actually only need Microsoft.Coyote package here. Thanks!
https://github.com/microsoft/coyote/blob/main/docs/get-started/install.md#installing-the-coyote-nuget-packages